examples/energy-gateway-app/commodity-metering/src/CommodityMeteringEventTriggers.cpp

```diff
@@ -35,8 +35,8 @@ static constexpr uint32_t TariffComponents2[] = { 0x5002, 0x5003, 0x5004 };
 
 // Non-constexpr storage for the actual data
 static const Structs::MeteredQuantityStruct::Type Data[] = {
-    { DataModel::List<const uint32_t>(TariffComponents1, sizeof(TariffComponents1) / sizeof(TariffComponents1[0])), 3500 },
-    { DataModel::List<const uint32_t>(TariffComponents2, sizeof(TariffComponents2) / sizeof(TariffComponents2[0])), -2000 }
+    { DataModel::List<const uint32_t>(TariffComponents1, MATTER_ARRAY_SIZE(TariffComponents1)), 3500 },
```

Comment:
I suspect this works:
```suggestion
    { DataModel::List(TariffComponents1), 3500 },
```